pkg/jobs/jobs.go

```diff
@@ -260,16 +260,6 @@ func (u Updater) started(ctx context.Context) error {
 			md.Payload.StartedMicros = timeutil.ToUnixMicros(u.now())
 			ju.UpdatePayload(md.Payload)
 		}
-		// md.RunStats can be nil because of the timing of version-update when exponential-backoff
-		// gets activated. It may happen that backoff is not activated when Update() function was
-		// called, which will cause to not populate md.RunStats. However, when the code reaches this
-		// point, version update may have been updated to enable backoff. In this case, we can skip
-		// updating num_runs and last_run, treating this job run as if backoff was not activated.
-		//
-		// TODO (sajjad): Update this comment after version 22.2 has been released.
-		if md.RunStats != nil {
-			ju.UpdateRunStats(md.RunStats.NumRuns+1, u.now())
-		}
```

Comment:
Note that we are still _reading_ last_run in a few places.  I haven't looked more closely at the call sites though, I think not setting this will mostly just impact logging.

Reply:
Removed.